[Sunbeam GA Launch] /index revamp

mtruj013 opened the pull request:
## Done

- Updated copy as per [doc](https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit#)
- NOTE: Will include [this CTA](https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit?disco=AAAAu0CfWYo) when the pdf becomes available 
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8039/
- See that the requested changes have been made

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3619

webteam-app commented:
Demo starting at https://microstack-run-231.demos.haus

juanruitina commented:
- Can you make sure to add "fresh" to "Get a **fresh** VM Ubuntu 22.04 LTS" under macOS?
- The "Fill in the survey" icon has some padding, [I adjusted the SVG here](https://github.com/canonical/microstack.run/assets/1409729/0fe43501-4f2c-413f-bf09-0a2d967d6048).
- "24x7" is barely readable on the fourth icon. Seems to be part of the pattern (max-height is set), but could we perhaps increase max-height of all four icons to 3rem?
- Off scope but small fix I hope: the logo in the navigation links to `#` both here and in /docs, so it doesn't really work. Can we have it make link to `\`?
- Navigation items take too much spaces and don't fit a single line in smaller screen sizes. I've made a suggestion for @tytus-kurek on the copy doc and asked for his OK.

mtruj013 commented:
Thanks @juanruitina, I believe I've addressed all of your comments if you could take another look?

juanruitina commented:
@tytus-kurek and I still need to agree on the label for the Discourse link on the nav, but other than that it looks good!

mtruj013 commented:
@julpal7

carkod reviewed (approved):
LGTM

julpal7 commented:
The [[ Download MicroStack Datasheet ]](https://ubuntu.com/engage/openstack-ebook) button is missing in the demo, but I see an issue with it has already been addressed in the copy doc, so I’m assuming it will get updated.

There is an unnecessary line under the header, it needs to be removed (it's more visible in the demo):
<img width="500" alt="Screenshot 2023-05-31 at 17 10 05" src="https://github.com/canonical/microstack.run/assets/132376509/c97bc5ff-1208-4852-952d-6835401ebfbc">

There is an HR divider missing between the sections here:
<img width="400" alt="Screenshot 2023-05-31 at 16 57 13" src="https://github.com/canonical/microstack.run/assets/132376509/adfd2853-9cd4-482b-9488-6a92db51a8db">

Between those sections as well:
<img width="400" alt="Screenshot 2023-05-31 at 17 02 56" src="https://github.com/canonical/microstack.run/assets/132376509/00e137ac-e30b-41a2-bb54-c134c32f4ebf">

I put comments on the dividers in the copy doc. Apart from those minor issues, everything looks okay.
@mtruj013 
Thanks!

mtruj013 commented:
Thanks @juanruitina and @julpal7 ! I've addressed your comments if you could both take another look? Julia I think the line under the suru is a vanilla bug specific to your browser, it isn't in the code and I can't see it locally or in the demo on either chrome or firefox

julpal7 commented:
@mtruj013 Everything looks good now. The line is showing for me in Chrome, but as long as it's not in the code, it must be a bug on my side.

ClementChaumel commented:
@julpal7 is your chrome zoomed in by any chance? I can see the line on both 110% and 90% zoom but not on 100%

julpal7 commented:
@ClementChaumel you're a genius! I was on 90% for some reason, I reset the zoom and it's gone

juanruitina commented:
LGTM!
